programs/bpf/rust/bignum/src/lib.rs

```diff
@@ -0,0 +1,130 @@
+//! @brief BigNumber Syscall test
+
+// #17082
```

Comment:
What's this?

Reply:
Fixed in a156563ebf54e8934b207cb12cb2d8aefb83b93d